display_list/effects/dl_image_filter.h

```diff
@@ -229,11 +229,15 @@ class DlBlurImageFilter final : public DlImageFilter {
   static std::shared_ptr<DlImageFilter> Make(SkScalar sigma_x,
                                              SkScalar sigma_y,
                                              DlTileMode tile_mode) {
-    if (SkScalarIsFinite(sigma_x) && sigma_x > SK_ScalarNearlyZero &&
-        SkScalarIsFinite(sigma_y) && sigma_y > SK_ScalarNearlyZero) {
-      return std::make_shared<DlBlurImageFilter>(sigma_x, sigma_y, tile_mode);
+    if (!SkScalarIsFinite(sigma_x) || !SkScalarIsFinite(sigma_y)) {
+      return nullptr;
     }
-    return nullptr;
+    if (sigma_x < SK_ScalarNearlyZero && sigma_y < SK_ScalarNearlyZero) {
+      return nullptr;
+    }
+    sigma_x = std::max(sigma_x, SK_ScalarNearlyZero);
```

Comment:
Actually, this is not correct. It basically takes a sigma of zero and makes it non-zero. I think the correct logic is:

`sigma_x = (sigma_x < NearlyZero) ? 0 : sigma_x;`

That's not really a "max" operation as the comparison and the assigned result are different.

Reply:
done